src/sagemaker_mxnet_container/serving.py

```diff
@@ -29,7 +29,7 @@
     'MXNET_CPU_WORKER_NTHREADS': '1',
     'MXNET_CPU_PRIORITY_NTHREADS': '1',
     'MXNET_KVSTORE_REDUCTION_NTHREADS': '1',
-    'MXNET_ENGINE_TYPE': 'NativeEngine',
+    'MXNET_ENGINE_TYPE': 'NaiveEngine',
```

Comment:
I'm curious how did this ever work?
And why would you set it to NaiveEngine rather than leave this blank.
See the [docs](https://mxnet.incubator.apache.org/versions/master/faq/env_var.html#engine-type) for details.

Reply:
When this was originally set for our 0.12 images, NaiveEngine was recommended for running inference against multiple models in parallel processes and performance testing backed up this decision.

However, there have been many versions since 0.12, so I'll remove the variable if you think that's better

Reply:
Yes, it is better to remove, unless you have hard data that it makes it better.
We ran lots of perf tests for MMS and it performed well without this env var, so MXNet defaulted to threaded engine. It may depend on the instance type though, but for now I suggest to remove it.

Reply:
sounds good, I'll remove it then